Automattic/jetpack
# Contact Form: replace jQuery UI select by native element

Fixes https://github.com/Automattic/jetpack/issues/34253

## Proposed changes:

The _Dropdown_ field of the _Contact Form_ is implemented with jQuery UI Selectmenu. This PR replaces it by a native select. Though we can't style the dropdown menu, this solution is more accessible, performant, and provides a better experience on mobile.

|Before|After|
|-|-|
|<img width="606" alt="screenshot-2023-12-04-at-6 15 48-pm" src="https://github.com/Automattic/jetpack/assets/1620183/4185ca4f-74ee-413b-85c4-1e43047c62a4">|<img width="616" alt="screenshot-2023-12-04-at-6 16 05-pm" src="https://github.com/Automattic/jetpack/assets/1620183/46e01532-ba73-4de9-a366-c7efe097731a">|

### Other information:


## Jetpack product discussion


pf5801-je-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Spin up a test site with this branch
- Create a new post
- Add the _Contact Form_ block to the page
- Add a Dropdown field and mark it as required
- Open the page preview
- Check that the Dropdown looks the same
- Check that the field is marked as invalid when submitting the form with an empty value
- Select an option
- Submit the form again and check that the results contain the field value
